### ISSUE TITLE
Ground truth pkl file for val & train

### ISSUE THREAD

**Issue by ankuPRK** [jagjeet-singh/argoverse-forecasting#1]
I was able to find the raw ground truth, but not the ground truth pkl file for the validation set to run the simple evaluator given in readme:

$ python eval_forecasting_helper.py --metrics --gt <path/to/ground/truth/pkl/file> --forecast <path/to/forecasted/trajectories/pkl/file> --horizon 30 --obs_len 20 --miss_threshold 2 --features <path/to/test/features> --max_n_guesses 6

Could you please share a link to the same, or guide to how to generate it with raw data, without generating features.

**Comment by jagjeet-singh** [comment]
Hi Prakhar,
The ground truth for train and val was not released explicitly, but it should be fairly easy to generate that. Just preprocess the raw csv files or the generated features file to have ground truth in the format:
```
gt: Dict[int, np.ndarray] = {seq_id: traj_for_future_3_secs}
Eg. gt = {1: np.array([[1,2],[1,3],[1,4],....]), 2: np.ndarray([100, 200], [101, 200], ...)}
```

**Comment by ankuPRK** [comment]
Okay got it, thanks!

**Comment by ankuPRK** [comment]
Hi, sorry for re-opening the issue, but I had another query. When we open the csv file for creating 'val' dataset, we get 50 points for the AGENT. Do we save all 50 (whole 5 secs) in the ground truth pkl file, or only last 30 (last 3 secs)?
